pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -604,6 +635,9 @@ def tk_to_qiskit(
         qcirc.add_register(qis_reg)
     symb_map = {Parameter(str(s)): s for s in tkc.free_symbols()}
     range_preds: Dict[Bit, Tuple[List["UnitID"], int]] = dict()
+
+    supported_gate_rebase.apply(tkc)
```

Comment:
Hmm interesting if I use ``RemoveRedundancies`` I get the following test failure

`FAILED qiskit_convert_test.py::test_cnry_conversion - assert [X node[1]; X node[2]; ] == [X node[0]; X...; X node[0]; ]
========================================================================= 1 failed, 22 passed, 13 warnings in 15.87s =========================================================================
(qiskit-env) callum@Callums-MacBook-Pro tests % pytest qiskit_convert_test.py
=================================================================================`
Will look into this more.

Reply:
Strangely this test failure disappeared after I changed to `tkc = tkcirc.copy()` , I am not sure why this would be.

Should this `test_cnry_conversion()` failure be investigated further or are you happy to proceed for now?

Reply:
Done, I have added the ``RemoveRedundancies`` pass after the Rebase.